src/cli/init/walkthrough.ts

```diff
@@ -84,13 +135,17 @@ ${packageJSON.print.replace(/^/gm, "    │   ")}
     ├── \x1b[1m${peprTSPath}\x1b[0m
     ├── \x1b[1m${readme.path}\x1b[0m
     └── \x1b[1m${tsConfig.path}\x1b[0m
-      `);
-
-  const confirm = await prompt({
-    type: "confirm",
-    name: "confirm",
-    message: "Create the new Pepr module?",
-  });
+  `;
 
-  return confirm.confirm;
+  if (skipPrompt !== undefined) {
+    return skipPrompt;
+  } else {
+    console.log(confirmationMessage);
+    const confirm = await prompt([confirmationPrompt]);
+    const shouldCreateModule =
+      confirm.confirm === "y" || confirm.confirm === "yes" || confirm.confirm === true
+        ? true
+        : false;
+    return shouldCreateModule;
```

Comment:
This _should_ be handled by `prompts.prompt()`'s `format` parameter, but I couldn't only ever get the unit _or_ journey tests to pass. So, I decided to just drop the validation logic here before the return. Pretty gross.